azooKeyMac/Windows/PromptInput/PromptInputView.swift

```diff
@@ -0,0 +1,529 @@
+                            endPoint: .trailing
+                        )
+                    )
+
+                Spacer()
+
+                Button(action: onCancel) {
+                    Image(systemName: "xmark")
+                        .foregroundColor(.secondary.opacity(0.8))
+                        .font(.system(size: 10, weight: .medium))
+                }
+                .buttonStyle(PlainButtonStyle())
+                .keyboardShortcut(.escape)
+            }
+            .padding(.horizontal, 12)
+            .padding(.top, 8)
+
+            // Input field with custom key handling
+            CustomTextField(
+                text: $promptText,
+                placeholder: "例: フォーマルにして",
+                isFocused: $isTextFieldFocused,
+                onSubmit: {
+                    if hoveredHistoryIndex != nil {
+                        // If history item is selected, use it and generate preview automatically
+                        promptText = getVisibleHistory()[hoveredHistoryIndex!].prompt
+                        hoveredHistoryIndex = nil
+                        isTextFieldFocused = true
+                        // Automatically request preview after setting the text
+                        DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
+                            requestPreview()
+                        }
+                    } else if showPreview {
+                        // Apply when Enter is pressed in preview mode
+                        onApply(previewText)
+                        onSubmit(promptText)
+                    } else {
+                        // Preview when Enter is pressed in input mode
+                        requestPreview()
+                    }
+                },
+                onDownArrow: {
+                    // Handle down arrow to start history navigation
+                    navigateHistory(direction: .down)
+                },
+                onUpArrow: {
+                    // Handle up arrow for history navigation
+                    navigateHistory(direction: .up)
+                }
+            )
+            .onChange(of: isTextFieldFocused) { isFocused in
+                // Notify parent window about focus changes
+                NotificationCenter.default.post(name: .textFieldFocusChanged, object: isFocused)
+            }
+            .onChange(of: promptText) { _ in
+                // When text field is edited after preview, hide preview and show history
+                if showPreview {
+                    withAnimation(.easeInOut(duration: 0.2)) {
+                        showPreview = false
+                        onPreviewModeChanged(false)
+                    }
+                }
+            }
+            .padding(.horizontal, 12)
+            .frame(height: 36)
+            .background(
+                RoundedRectangle(cornerRadius: 8)
+                    .fill(.regularMaterial)
+            )
```

Comment:
あんま細かいとこまで見るの無理だから気づいたとこだけ言うけど、`.background()`はdeprecated
```suggestion
            .background {
                RoundedRectangle(cornerRadius: 8)
                    .fill(.regularMaterial)
            }
```